lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -102,11 +152,16 @@ public static int binarySearch(int[] values, int start, int end, int value) {
     int pivotIndex = (start + end) / 2; // The index in the middle of the array.
 
     // TODO(you): Finish implementing this algorithm
-
+    if (values[pivotIndex] == value) {
+      return pivotIndex;
+    } else if (values[pivotIndex] > value) {
+      return binarySearch(values, start, pivotIndex - 1, value);
```

Comment:
Another challenge: implement this method to use iterative approach instead of the recursive one.